modules/parent-join/src/internalClusterTest/java/org/elasticsearch/join/query/ChildQuerySearchIT.java

```diff
@@ -1404,6 +1404,7 @@ public void testParentChildQueriesViaScrollApi() throws Exception {
 
         for (QueryBuilder query : queries) {
             assertScrollResponsesAndHitCount(
+                client(),
```

Comment:
I updated the `assertScrollResponsesAndHitCount`  test helper method so that it will work with the ESSingleNodeTestCase, and thus make it available for testing the metric changes.  Hence this seemingly unrelated change.